README.md

```diff
@@ -41,8 +41,7 @@ Brings up app at localhost:3000 with some test data:
 
 ```
 gem install foreman
-docker compose up -d
-docker compose stop web
+docker compose up -d sdr-api techmd
```

Comment:
Please move to a separate PR for discussion

Reply:
Dropped the commit